uktrade/data-hub-frontend
# Salary range disabled

## Description of change

Change the average salary edit/create field in an investment. Salary ranges that have been disabled before the creation of the investment project can no longer be selected for the investment. However for existing investments with a salary range that is disabled before the project was created, it should still be displayed but can no longer be selected. Still can be selected in django admin 

## Test instructions

The salary range that is disabled before the project was created is no longer displayed on the page. Go to django page and disable a salary range before an investment project is created. Go on to the investment page and edit the salary range, the range should no longer be an option to select

## Screenshots

### Before

<img width="698" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/15215184/0156325c-9023-4e9a-962d-3731f356e348">

<img width="965" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/15215184/bf15c434-7943-42c8-afce-118bef0b06d9">

<img width="965" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/15215184/7ea5c0e5-7294-47f5-beaa-3d99066115fb">



### After
<img width="698" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/15215184/f2a64708-562f-464f-b0af-b108c4ab709d">

Disabled before

<img width="922" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/15215184/6293dd18-cda6-45f8-8c4e-048aefa2bc5c">

<img width="984" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/15215184/94aaa2e1-f4f8-4b86-8211-bd13a240c31e">

Disabled after

<img width="922" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/15215184/278604b6-ae78-4084-8220-979af154c20f">

<img width="965" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/15215184/e74212aa-4a91-4043-9884-774770b8aea5">


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
